src/views/bootablevolumes/actions/components/DeleteBootableVolumesModal.tsx

```diff
@@ -27,12 +27,12 @@ const DeleteBootableVolumesModal: FC<DeleteBootableVolumesModalProps> = ({
       obj={dataSource}
       isOpen={isOpen}
       onClose={onClose}
-      headerText={t('Delete label?')}
-      onDeleteSubmit={deletePreferenceLabel(dataSource)}
+      headerText={t('Delete labels?')}
+      onDeleteSubmit={deleteBootableVolumeMetadata(dataSource)}
       bodyText={
         <Trans t={t}>
-          Please note that only the label data will be deleted and that the bootable volume will
-          remain. Are you sure you want to delete the label for bootable volume{' '}
+          Please note that only the labels data will be deleted and that the bootable volume will
```

Comment:
If I understand correctly, this action will delete both the description annotation and the default labels. Shouldn't the message indicate that the description annotation will be deleted as well?

Reply:
This is how it was designed:
https://docs.google.com/document/d/15Cv54978AZKeukkurvkFN1YEx_kBsm_3XcmLey_cgZY/edit#heading=h.2mig9v8s38t
Note that,unfortunately, not everything is up to date in that design doc. I asked Foday to update it but it seems it takes time 🤷🏼‍♀️

Reply:
Understood. Let's merge it for now and we can readdress it when the design gets updated.